### PR TITLE
Add svn move info to releasing.md

### PR DESCRIPTION
**Pull request by pnoltes**
This PR has some small text improvements and adds a example `svn move` command that can be used to move voted release artifacts from the ASF dev to release area.

### CODE REVIEW

**Review by PengZheng — APPROVED** [review]
LGTM

File: source/contributing/releasing.md [hunk 1]
```diff
@@ -318,20 +331,23 @@ The following template must be used:
 To: announce@apache.org, dev@celix.apache.org
 Subject: [ANNOUNCE] Apache Celix X.Y.Z released
 
-The Apache Celix team is pleased to announce the release of Celix X.Y.Z.
 
-Celix is an implementation of the OSGi specification adapted to C. It will follow the API as close as possible, but since the OSGi specification is written primarily for Java, there will be differences. An important aspect of the implementation is interoperability between Java and C. This interoperability is achieved by porting and implementing the Remote Services specification in Celix.
+The Apache Celix team is pleased to announce the release of Celix 2.4.0.
```
Comment (rlenferink) [inline source/contributing/releasing.md]: This was `X.Y.Z`  , why did you change this to `2.4.0` ?

I also think this should be `Apache Celix X.Y.Z`, instead of just `Celix X.Y.Z`

**Review by rlenferink — APPROVED** [review]
